batch_job/src/work_queue_factory.c

```diff
@@ -771,7 +778,7 @@ int read_config_file(const char *config_file) {
 	last_time_modified = new_time_modified;
 	fprintf(stdout, "Configuration file '%s' has been loaded.", config_file);
 
-	fprintf(stdout, "master-name: %s\n", project_regex);
+	fprintf(stdout, "manager-name: %s\n", project_regex);
```

Comment:
Breaks config file?

Reply:
It does not. The config file reads both, with preference to manager-name.